pkg/agent/sysadvisor/plugin/poweraware/metric/error_code.go

```diff
@@ -0,0 +1,29 @@
+/*
+Copyright 2022 The Katalyst Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package metric
+
+const (
+	ErrorCodeInitFailure                  = 9505001
```

Comment:
Why don't we emit these as strings that describe the error?

Reply:
ok will come up with error string list (which is much human friendlier) replacing the numbers